travis.sh

```diff
@@ -287,9 +295,10 @@ function test_workspace() {
 
    # Show test results summary and throw error if necessary
    catkin_test_results || exit 2
+ }
 
-   # Copy build products for code coverage reporting
-   [[ "${TEST:=}" == *code-coverage* ]] && travis_run_simple cp -r $ROS_WS/build /root/$REPOSITORY_NAME
+function send_codecov_report() {
```

Comment:
This is a one-line function that is only used once. Please move content to the call location.